portfolio/src/main/webapp/chart.html

```diff
@@ -1,12 +1,25 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <title>Google Virtual Zoo Population</title>
-  <script src="https://www.gstatic.com/charts/loader.js"></script>
-  <script src="script.js"></script>
-</head>
-<body onload="loadChartsApi()">
-  <h1>Google Virtual Zoo Population</h1>
-  <div id="chart-container"></div>
-</body>
+  <head>
+    <title>Google Virtual Zoo Population</title>
+    <script src="https://www.gstatic.com/charts/loader.js"></script>
+    <script src="script.js"></script>
+  </head>
+  <body onload=loadChartsApi();loadChartsApi2();>
+    <h1>Amazon Stock Price 2014-2018</h1>
+    <div id="chart-container"></div>
+    <h1>Favorite Seasons</h1>
+    <div id="chart2-container"></div>
+    <h2>Vote</h2>
+    <form action="/season-data" method="POST">
```

Comment:
Is there a `/season-data` servlet that needs to be added to this PR?  When I tried this code I got a 404 error after clicking the submit button below

Reply:
addressed in "added seasons servlet". On my end everything functions fine even in the absence of the servlet, which is odd.